### DISCUSSION TITLE
Discord + Cardano wallet integration

### DISCUSSION THREAD

**Discussion by wlstylr** [SecretDecks/Documentation#105] [gov.channel]
Quasar recently introduced me to CardanoCon.  It is a discord bot that allows you to connect the Nami wallet (browser based Cardano wallet) so that you can to do things like synchronize roles on discord by asset held in wallet and mint NFTs through bot commands.  

We may be able to use this as part of our off to on-chain governance pilot.  For example, we could mint voting NFTs based on SourceCred data, and then only those with voting NFTs in their wallet would have access to the voting channel on discord.   After the vote is completed, a screenshot of the voting results could be taken and minted as an NFT all in the Discord server. 

Please take a look when you have a chance and let's see if we can come up with some more ideas @stephen-rowan @DanM3rcurius @tamagit80 @evolayak 

https://cardanocon.io/features

**Comment by QuasarChains** [comment]
I would be happy to sponsor the discord bot for a mmonth.
https://cardanocon.io/features/payment

**Reply by wlstylr** [reply]
Thanks!  That is very kind offer!  I think right now we want to start with brainstorming some potential use cases that would fit into what we are currently working on in regards to governance and contribution.   Let's get some ideas flowing:)

**Reply by QuasarChains** [reply]
The cardanocon bot also allows for roles to be issued and synchronized through tokens which could an interesting method to explore. I believe there is a way to use this feature as another method to enhance the contributions, and perhaps an avenue to assist with maintaining focus. 

https://cardanocon.io/features/synchronization

**Comment by DanM3rcurius** [comment]
After testing the flow our selves, we could think about if this could be employed in the sprint voting approval?

Important question we have to ask our selves: What does it do better for us now that we can't get done with other tools?
